test/OpenCL.std/printf.spvasm

```diff
@@ -0,0 +1,66 @@
+; REQUIRES: spirv-as
+; RUN: spirv-as --target-env spv1.0 -o %t.spv %s
+; RUN: llvm-spirv %t.spv -r --spirv-target-env=SPV-IR -o - | llvm-dis | FileCheck %s --check-prefixes=CHECK,CHECK-SPV-IR
```

Comment:
Could you please add checks that **SPIRV -> SPIRV-IR -> SPIR-V** round trip is working correctly? I.e. all instructions in produced SPIR-V friendly IR are recognized by SPIRVWriter?

Reply:
Sure, I'll prepare such translation testing.

Reply:
Tests for SPIRV-IR -> SPIRV translation implemented.